src/pages/login/login.js

```diff
@@ -49,6 +49,26 @@ const Login = () => {
       .matches(formRegExp.password, errorMessages[language].value.password)
       .required(' ')
   });
+  const auth = () => {
+    const _onInit = (auth2) => {
+      console.log('init OK', auth2);
+    };
+    const _onError = (err) => {
+      console.log('error', err);
+    };
+    window.gapi.load('auth2', function () {
+      window.gapi.auth2
+        .init({
+          client_id:
+            '648273081193-f3b5j2al2k4tu6hgm0ogdci5ag431pnc.apps.googleusercontent.com'
```

Comment:
Please, move these creds to the config file

Reply:
Fixed